stock_ownership_by_move/view/picking.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+  <data>
+
+    <record id="view_picking_form" model="ir.ui.view">
+      <field name="name">view_picking_form</field>
+      <field name="model">stock.picking</field>
+      <field name="inherit_id" ref="stock.view_picking_form"/>
+      <field name="arch" type="xml">
+
+        <xpath expr="//field[@name='owner_id']/.." position="replace"/>
```

Comment:
It is a div, containing a label, a field and a button. I don't think I can hide it.